src/velero/chart/templates/uds-package.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: uds.dev/v1alpha1
+kind: Package
+metadata:
+  name: velero
+  namespace: {{ .Release.Namespace }}
+spec:
+  network:
+    allow:
+      # Todo: wide open for now for pushing to s3
+      - direction: Egress
+        podLabels:
+          app.kubernetes.io/name: velero
+        remoteGenerated: Anywhere
+
+      - direction: Egress
+        podLabels:
+          batch.kubernetes.io/job-name: "velero-upgrade-crds"
+        remoteGenerated: KubeAPI
+
+      - direction: Ingress
+        remoteNamespace: monitoring
+        remoteSelector:
+          app: prometheus
+        selector:
+          app.kubernetes.io/name: velero
+        port: 8068
+        description: "Prometheus Metrics"
+
+      - direction: Ingress
+        selector:
+          app.kubernetes.io/name: velero
+        remoteNamespace: promtail
+        remoteSelector:
+          app.kubernetes.io/name: promtail
+        ports:
+          - 8080
+        description: "Promtail Log Storage"
+
+      - direction: Egress
+        remoteNamespace: tempo
+        remoteSelector:
+          app.kubernetes.io/name: tempo
+        port: 9411
+        description: "Tempo"
+        
```

Comment:
```suggestion

```
I feel bad for missing this earlier but I don't think either of these are actually needed. The promtail one seems like maybe a copy/paste from somewhere it was necessary? Tempo I lean towards leaving off since it isn't in core yet (tracking that this is in other packages though). If/when we add Tempo it would be autogenerated and added to all packages since its necessary everywhere.